storage/src/main/java/org/apache/kafka/storage/internals/log/LogValidator.java

```diff
@@ -310,7 +304,6 @@ public ValidationResult assignOffsetsNonCompressed(LongRef offsetCounter,
             now,
             records,
             maxTimestamp,
-            shallowOffsetOfMaxTimestamp,
```

Comment:
`shallowOffsetOfMaxTimestamp` is useless, so could you please remove it also?

Reply:
Removed it. Thanks for the reminder.